src/index.html

```diff
@@ -1,13 +1,350 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>B&O | Media</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.png"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-block">
+            <a href="#" class="logo">
+              <img
+                src="./images/logo.svg"
+                alt="B&O logo"
+                class="logo__image"
+              >
+            </a>
+
+            <div class="header__icons">
+              <a href="tel:+1 234 5555-55-55" class="header__call icon icon--phone"></a>
+              <a href="#page__menu" class="icon icon--menu"></a>
```

Comment:
Need to fix format phone. Fix it everywhere
```suggestion
            <div class="header__icons">
              <a href="tel:+123455555555" class="header__call icon icon--phone"></a>
              <a href="#page__menu" class="icon icon--menu"></a>
```